src/services/products/index.ts

```diff
@@ -0,0 +1,12 @@
+import { ProductResponse } from 'components/pages/Products/Products.types'
+import { productsAPI } from 'config/axios'
+
+export const getProducts = async (
+  page: number,
+  perPage = 6,
+): Promise<ProductResponse> => {
+  const response = await productsAPI.get(
+    `api/products?perPage=${perPage}&page=${page}`,
+  )
+  return response.data
+}
```

Comment:
Actually you don't have to explicitly declare a return type, you can put generic to get function and it will infer the type :D